src/models/tripData.js

```diff
@@ -0,0 +1,8 @@
+export default class TripData {
+  constructor(isFirstTrip = false) {
+    this.id = Date.now().toString();
+    this.startDate = null
+    this.endDate = null
+    this.isFirstTrip = isFirstTrip;
+  }
+}
```

Comment:
Yeah it felt a bit Object Orienty so maybe it's overkill, it would be good to see a more JS way. This kind of stemmed from my original idea when I was working on this in rails, the idea being that each new trip line added would have a corresponding `Trip` model for when it came to running the calculation.